package/service/session_service.go

```diff
@@ -98,25 +98,34 @@ func (s *SessionServiceImpl) ValidateSession(tx *gorm.DB, sessionId string) (sch
 	if err != nil {
 		s.logger.Errorf("Error getting session by id: %v", err.Error())
 		if err == gorm.ErrRecordNotFound {
-			return schemas.ValidateSessionResponse{Valid: false, UserId: -1}, ErrSessionNotFound
+			return schemas.ValidateSessionResponse{Valid: false, User: schemas.User{Id: -1}}, ErrSessionNotFound
```

Comment:
Pleaser create const invalid user, and reuse it. This will make testing easier and overall enchance code readability